src/components/Asset/AssetContent/MetaFull.tsx

```diff
@@ -7,13 +7,14 @@ import { useWeb3 } from '@context/Web3'
 import { Asset, Datatoken, LoggerInstance } from '@oceanprotocol/lib'
 
 export default function MetaFull({ ddo }: { ddo: Asset }): ReactElement {
-  const [paymentCollector, setPaymentCollector] = useState<string>()
+  const [paymentCollector, setPaymentCollector] = React.useState<string>()
```

Comment:
Our pattern is to always import React's methods directly, so in this case `useState` and not `React.useState`

Reply:
I used that to make the tests work, as `React.useState = jest.fn().mockReturnValue()` works but `useState = jest.fn().mockReturnValue()` doesn't. Do you have some advice on how to make the mock work without `React.useState`?

Reply:
rather trigger whatever leads to updating that state instead of trying to mock it directly. So in this case you should not mock the state but the method which runs before state is updated, so `datatoken.getPaymentCollector()`